Dockerfile

```diff
@@ -28,12 +28,17 @@ RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get inst
 WORKDIR /root
 
 ### JDK
-RUN wget https://download.java.net/java/GA/jdk17.0.2/dfd4a8d0985749f896bed50d7138ee7f/8/GPL/openjdk-17.0.2_linux-x64_bin.tar.gz -P /tmp
-RUN tar xvf /tmp/openjdk-17.0.2_linux-x64_bin.tar.gz -C /
+RUN wget https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz -P /tmp
+RUN tar xvf /tmp/openjdk-11.0.2_linux-x64_bin.tar.gz -C /
```

Comment:
We are downgrading jdk version from 17.0.2 to 11.0.2. Will there be any security concern? 🤔

Reply:
Java 17 is supported by Gradle version 7. We could bump this up to java 13 since `gradle 6` is compatible with it.

Here is a link to the gradle and jdk compatibility matrix. https://docs.gradle.org/current/userguide/compatibility.html

Please let me know if you want this bumped

Reply:
jdk 11 is supported till 2026 - that should give time for projects using older gradles to upgrade and we can then pull support for the older version of java / gradles